content/en/news/releases/1.6.x/announcing-1.6/_index.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Announcing Istio 1.6
+linktitle: 1.6
+subtitle: Major Update
+description: Istio 1.6 release announcement.
+publishdate: 2020-05-19
+release: 1.6.0
+skip_list: true
+aliases:
+    - /news/announcing-1.6.0
+    - /news/announcing-1.6
+---
+
+We are pleased to announce the release of Istio 1.6!
+
+{{< relnote >}}
+
+With this release, we are continuing the path we charted earlier this year in
+our [roadmap post](/blog/2020/tradewinds-2020/), sailing toward more
+simplicity, a better installation experience, and some other goodies as well.
+
+Here’s some of what’s coming to you in today's release:
+
+## Simplify, simplify, simplify
+
+Last release, we introduced **Istiod**, a component that reduced the number of
+components in an Istio installation by combining the functionality of several
+other components. In Istio 1.6, you'll find that Citadel, the sidecar
```

Comment:
nit: To the unfamiliar this may sound like we are removing more things in 1.6 as part of the default install. For most users these were already removed in 1.5. Now there is just no way to configure them at all.

Maybe we can rephrase this to something along the lines of "In Istio 1.6 we have completed this transition and have fully moved functionality into Istiod, allowing us to remove x y z" or similar?

Reply:
Great suggestion. Done.